Automatic Validation ended with:
> 2025-02-05T15:43:05.8653658Z ##[error] Manifest Error: The multi file manifest has inconsistent field values. [PackageVersion] Value: 3.13.1 File: Python.Python.3.13.locale.zh-CN.yaml Manifest Error: The multi file manifest has inconsistent field values. [ManifestVersion] Value: 1.6.0 File: Python.Python.3.13.locale.zh-CN.yaml
 2025-02-05T15:43:05.8703673Z ##[error] Manifest Error: The multi file manifest has inconsistent field values. [PackageVersion] Value: 3.13.1 File: Python.Python.3.13.locale.zh-CN.yaml Manifest Error: The multi file manifest has inconsistent field values. [ManifestVersion] Value: 1.6.0 File: Python.Python.3.13.locale.zh-CN.yaml

(Automated response - build 977.)

--- a/manifests/p/Python/Python/3/13/3.13.2/Python.Python.3.13.locale.zh-CN.yaml
+++ b/manifests/p/Python/Python/3/13/3.13.2/Python.Python.3.13.locale.zh-CN.yaml
@@ -1,5 +1,5 @@
 # Created with YamlCreate.ps1 Dumplings Mod
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.6.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.9.0.schema.json
 
 PackageIdentifier: Python.Python.3.13
 PackageVersion: 3.13.1
@@ -32,4 +32,4 @@ Documentations:
 - DocumentLabel: 文档
   DocumentUrl: https://www.python.org/doc/
 ManifestType: locale
-ManifestVersion: 1.6.0
+ManifestVersion: 1.9.0

--- a/manifests/p/Python/Python/3/13/3.13.2/Python.Python.3.13.locale.zh-CN.yaml
+++ b/manifests/p/Python/Python/3/13/3.13.2/Python.Python.3.13.locale.zh-CN.yaml
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.9.0.schema.json
 
 PackageIdentifier: Python.Python.3.13
-PackageVersion: 3.13.1
+PackageVersion: 3.13.2
 PackageLocale: zh-CN
 Publisher: Python Software Foundation
 PublisherUrl: https://www.python.org/

